packages/framework/build.xml

```diff
@@ -106,6 +106,14 @@
             <arg value="assets:install"/>
             <arg value="${path.root}/web/"/>
         </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="elfinder:install"/>
+        </exec>
+        <exec executable="${path.php.executable}" passthru="true" checkreturn="true">
+            <arg value="${path.bin-console}"/>
+            <arg value="shopsys:elfinder:postinstall"/>
+        </exec>
```

Comment:
I would move this to `composer.json` -> `scripts` -> `auto-scripts`

Reply:
the commands to install elfinder have to be after assets:install because it deletes folder where the elfider is installed